__tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap

```diff
@@ -146,11 +146,6 @@ exports[`LibraryContainer Test UI rendering of Library Item and child components
         onError={[Function]}
         src="test-file-stub"
       />
-      <img
-        className="LibraryItemIcon"
-        onError={[Function]}
-        src=""
-      />
       <div
```

Comment:
@mjkkirschner @ramramps these tags are no longer generated since the `none` element type is no longer adding icons so I have to update the snapshot to exclude them.  If they were to get generated the test will now fail.  You can see the tests that generate these 2 snaps [here](https://github.com/DynamoDS/librarie.js/blob/master/__tests__/Snapshottest/UIOutputComparisonTests.tsx#L41).

Reply:
Dynamo dynamically generates the data in the libraryUI so it is hard ensure all the required items are loaded in Librarie, for example if I wanted to verify icons are still generated for the nodes.  I am not sure if this is adequate or if there is a better approach I could be taking here.